addons/spreadsheet/static/tests/global_filters/global_filters_model_test.js

```diff
@@ -1318,6 +1334,35 @@ QUnit.module("spreadsheet > Global filters model", {}, () => {
         assertDateDomainEqual(assert, "date", "2016-05-18", "2019-05-17", computedDomain);
     });
 
+    QUnit.test("from_to date filter at model loading", async function (assert) {
+        const model = new Model({
+            globalFilters: [
+                { type: "date", rangeType: "from_to", label: "From To", fields: {}, id: "1" },
+            ],
+        });
+        assert.deepEqual(model.getters.getGlobalFilterValue("1"), { from: undefined, to: undefined });
+    });
+
+    QUnit.test("from_to date filter domain value", async function (assert) {
```

Comment:
also test with only from, only to
add a test where we clear the value (CLEAR_GLOBAL_FILTER_VALUE)